src/bolt/events/RegisterEventAppMention.ts

```diff
@@ -5,18 +5,22 @@ import {
 } from '../utils';
 import {
     getDataRetentionInMonth,
+    getDefaultNumberOfApproval,
+    getDefaultNumberOfReview,
     getDefaultReviewChannel,
     getGroupToMentionInChannel,
-    getRepositoryNumberOfApprovals,
-    getRepositoryNumberOfReviews,
+    getRepositoryNumberOfApproval,
+    getRepositoryNumberOfReview,
```

Comment:
correct grammar would be to keep this plural, why change it?

Reply:
Old habit of using singular for all variable naming.